Assets/Scripts/DDAA/Variables/EnemyMeleeDamageDDAA.cs

```diff
@@ -34,11 +34,12 @@ public static EnemyMeleeDamageDDAA Instance
     // All of the parameters below are the ones to change, when adjusting the DDA (unless there's a bug)
 
     //static parameters
-    private static readonly float baseMeleeDamagePoint = 10f;
-    public static readonly float minMeleeDamage = 45f;
+    private static readonly float baseMeleeDamagePoint = 9f;
+    public static readonly float minMeleeDamage = 30f;
     public static readonly float maxMeleeDamage = 100f;
     private static readonly float dpgContribution = 0.3f;
     private static readonly float meleeDamagePointContribution = 3f;
+    //actual value = minMeleeDamage + (baseMeleeDamagePoint * meleeDamagePointContribution) + (difficultiesPointGlobal(10 by default) * dpgContribution);
 
```

Comment:
?

Reply:
just a note we left for ourselves to know how the standard value is calculated